lib/mssql.js

```diff
@@ -16,12 +16,16 @@ exports.initialize = function initializeSchema(dataSource, callback) {
   debug('Settings: %j', settings);
   var driver = new MsSQL(settings);
   dataSource.connector = driver;
-  driver.connect(function(err, connection) {
-    dataSource.client = connection;
-    dataSource.connector.dataSource = dataSource;
-    dataSource.connector.tableNameID = dataSource.settings.tableNameID;
-    callback && callback(err, connection);
-  });
+  if (settings.lazyConnect) {
+    return true
+  } else {
+      driver.connect(function(err, connection) {
+      dataSource.client = connection;
+      dataSource.connector.dataSource = dataSource;
+      dataSource.connector.tableNameID = dataSource.settings.tableNameID;
```

Comment:
It won't be necessary to execute the 3 lines above if we set up connector.dataSource/tableNameID before hand.